backend/backend/settings/celery.py

```diff
@@ -0,0 +1,100 @@
+import os
+from typing import Any, Optional
+from urllib.parse import quote_plus
+
+from backend.constants import FeatureFlag
+
+# Requires PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python env setting
+# to be loaded prior.
+from unstract.flags.feature_flag import check_feature_flag_status
+
+missing_settings = []
+
+
+def get_required_setting(
```

Comment:
@hari-kuriakose we need to check missing_settings and raise error. Else using the `get_required_setting` wouldn't have any effect. Reference: https://github.com/Zipstack/unstract/blob/main/backend/backend/settings/base.py#L532-L536